src/components/calendar/CalendarView.jsx

```diff
@@ -33,29 +34,28 @@ const CalendarView = (props) => {
 
   // performs search with new "from" and "to" dates
   useEffect(() => {
-    axios.get(`${process.env.NEXT_PUBLIC_DOMAIN}/api/events?lng=${searchObj.lng}&lat=${searchObj.lat}&from=${currentStartDate}&to=${currentEndDate}&sort=time`)
+    axios.get(`${process.env.NEXT_PUBLIC_DOMAIN}/api/events?lng=${searchObj.lng}&lat=${searchObj.lat}&from=${fakeStartDate}&to=${fakeEndDate}&sort=time`)
       .then(res => {
-        // do we need a state for queryData?
-        setQueryData(res.data);
         return res.data;
       })
       .then(data => {
         const tempEventDates = new Set();
         data.features.forEach(event => {
-          const hasKeywords = searchObj.keywords.split(' ').every(keyword => {
-            if (event.properties.name.includes(keyword)
-              || event.properties.buskerName.includes(keyword)
-              || event.properties.description.includes(keyword)) {
+          const hasKeywords = searchObj.keywords.toLowerCase().split(' ').every(keyword => {
+            if (event.properties.name.toLowerCase().includes(keyword)
+              || event.properties.buskerName.toLowerCase().includes(keyword)
+              || event.properties.description.toLowerCase().includes(keyword)) {
               return true;
             }
             return false;
           });
           if (hasKeywords) {
-            tempEventDates.add(parseInt(event.properties.starts.slice(8, 10), 10));
+            tempEventDates.add(
+              (new Date(event.properties.starts.toString().slice(0, 10))).toString().slice(4, 15),
```

Comment:
This is not a good way to manipulate dates. The Date object has [its own methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) that don't require stringifying objects, cutting the strings up, and parsing them back into objects.

If you wanted to write a function that multiplies a number by negative 1, you could have it stringify the input, put "-" at the start of the string or delete it from the string if it's already there, and then parse the string back into a date. You could do that! It would work. But you really shouldn't.